features/standard/ContentTree.feature

```diff
@@ -0,0 +1,28 @@
+@IbexaOSS @IbexaHeadless @IbexaCommerce @IbexaExperience @javascript
+
+Feature: Content tree basic operations
+  Scenario: Content tree can be displayed
+    Given I am logged as admin
+    When I'm on Content view Page for "root"
+    Then I verify Content tree visibility
+
+  Scenario: It is possible to display items on Content tree
+    Given I create "article" Content items
+      | title       | short_title   | parentPath | language |
+      | Article1    | art1          | root       | eng-GB   |
+      | Article2    | art2          | root       | eng-GB   |
+      | Article3    | art3          | root       | eng-GB   |
+    And I am logged as admin
+    And I'm on Content view Page for "root/art1"
```

Comment:
Could you please add here a dedicated step to verify items on the content tree? 
`And I'm on Content view Page for "root/art1"` does not verify tree (only main container, breadcrumbs, content name and content type). We need something similar to https://github.com/ibexa/taxonomy/blob/5570a47c15d685381ca6603e27dba6e0a60800d1/features/Taxonomy.feature#L40